content/sensu-go/5.17/installation/install-sensu.md

```diff
@@ -144,19 +144,43 @@ For a complete list of configuration options, see the [backend reference][6].
 
 ### 3. Initialize
 
-_**NOTE**: If you are using Docker, skip this step and continue with [4. Open the web UI][36]. The `sensu-backend init` command is not implemented for Docker._
-
 **With the backend running**, run `sensu-backend init` to set up your Sensu administrator username and password.
 In this initialization step, you only need to set environment variables with a username and password string &mdash; no need for role-based access control (RBAC).
 
-Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use.
 
-{{< highlight shell >}}
+{{< language-toggle >}}
+
+{{< highlight Docker >}}
+docker run \
+-e SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME \
+-e SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD \
+--restart always \
+--name sensu \
+sensu/sensu:latest \
+sensu-backend start \
```

Comment:
Should `sensu-backend start \` be removed here?

Reply:
It needs to stay - we don't need line 162: `sensu-backend init` as it's called in the Dockerfile with some logic to only run if the container is being launched as a backend.

https://github.com/sensu/sensu-enterprise-go/blob/fix/dockerfile-init-user-settable-admin/docker-scripts/sensu-entrypoint.sh#L21

Reply:
Paging @amdprophet